database/migrations/sqlite/1717749007994908_releases.up.sql

```diff
@@ -4,8 +4,17 @@ CREATE TABLE IF NOT EXISTS releases
     releaseVersion INTEGER, -- release ID given by the client that calls CreateApplicationVersion
```

Comment:
Let's also add for PostgreSQL?

Reply:
it's just a symlink, so it didn't need any changes. But it exists already.

Reply:
oh I see, the symlink already existed before so I didn't see it in the PR diff.